src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_commands.py

```diff
@@ -189,6 +189,7 @@ def load_flexibleserver_command_table(self, _):
                                  setter_name='flexible_server_update_set', setter_type=flexible_server_custom_common,
                                  setter_arg_name='parameters',
                                  custom_func_name='flexible_server_update_custom_func')
+        g.custom_command('upgrade', 'flexible_server_version_upgrade', custom_command_type=flexible_server_custom_common)
```

Comment:
Why do we need a new command `upgrade`? Can we just add a new parameter for existing `update` command?

Reply:
We decided with feature owner @qime that the best option would be `upgrade` instead of `update` because `upgrade` can do potentially irreversible changes, so we try to reduce the risk of user errors.